src/blocks/block-coauthor-avatar/block.json

```
@@ -0,0 +1,55 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "co-authors-plus/avatar",
+	"version": "1.0.0",
+	"title": "Co-Author Avatar",
+	"category": "theme",
+	"description": "Displays a small scale version of a co-author's avatar. Utilizes fallbacks from Gravatar so everyone has an avatar.",
```

@douglas-johnson Minor, can we add a [`coauthors` keyword](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/#keywords) for the coauthors subblocks?

```suggestion
	"description": "Displays a small scale version of a co-author's avatar. Utilizes fallbacks from Gravatar so everyone has an avatar.",
	"keywords": [ "coauthors" ],
```

I noticed in testing that typing `/coauthors` into the editor only pulls up the main coauthors block, and a dash is needed to show the rest. I understand why, but it feels more ergonomic to show for both.

---

Addressed in https://github.com/Automattic/Co-Authors-Plus/pull/997/commits/3a54eb7dca3ecc82e0ee4b0e820da0c258d158b2